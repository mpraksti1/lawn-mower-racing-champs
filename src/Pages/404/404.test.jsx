import ErrorPage from '.';

describe('404 component', () => {
  it('renders without crashing', () => {
    shallow(<ErrorPage />);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<ErrorPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
