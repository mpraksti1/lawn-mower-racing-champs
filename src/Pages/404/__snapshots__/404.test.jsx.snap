// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 component matches snapshot 1`] = `
<Centered>
  <Text>
    Whoops! No page to be found here.
  </Text>
</Centered>
`;
